runners/samza/build.gradle

```diff
@@ -110,6 +110,8 @@ def sickbayTests = [
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testEncodingIOException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingNPException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingIOException',
+        // https://issues.apache.org/jira/browse/BEAM-6354
+        'org.apache.beam.sdk.io.BoundedReadFromUnboundedSourceTest.testTimeBound',
```

Comment:
Is there an example of this failing in CI/how did we discover this issue? If there is, could you link it in the PR description?

Reply:
It was reported here [BUG21709](https://github.com/apache/beam/issues/21709) and the CI error is here [Failed BoundedReadFromUnboundedSourceTest.testTimeBound](https://ci-beam.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/9581/testReport/org.apache.beam.sdk.io/BoundedReadFromUnboundedSourceTest/testTimeBound/)